### PR TITLE
Gantt chart: misc fixes and documentation 

### PR DESCRIPTION
**Pull request by sappelhoff**
cc @Remi-Gau 

will add more commits to add some documentation in the `tools/README.md`.

We should also update the BIDS spec release protocol, so that upon a release, the `timeline.csv` and `beps_completed.yml` files are updated 🤔

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Attribution 4.0 International
 
-Copyright (c) 2020-2023, BIDS Contributors.
+Copyright (c) 2020-2024, BIDS Contributors.
```
Comment (Remi-Gau) [inline LICENSE:3]: From what @effigies once told me, we can just delete the date part of this.

Can't find the link anymore but it is not something that actually matters.

**Review by Remi-Gau — APPROVED** [review]
Good with me

File: LICENSE (hunk 1 above)
Comment (effigies) [reply LICENSE:3]: https://hynek.me/til/copyright-years/

Comment (sappelhoff) [reply LICENSE:3]:
```suggestion
Copyright (c) 2020, BIDS Contributors.
```

**Pull request merged**